File: TripKitAndroidUI/src/main/java/com/skedgo/tripkit/ui/geocoding/CitySelectedEvent.java
package com.skedgo.tripkit.ui.geocoding;
import com.skedgo.tripkit.ui.data.places.LatLngBounds;
import org.immutables.value.Value;

@Value.Immutable(builder = false)
public interface CitySelectedEvent {
  @Value.Parameter
  LatLngBounds bounds();
}

File: TripKitAndroidUI/src/main/java/com/skedgo/tripkit/ui/geocoding/CitySelectedEvent.kt
package com.skedgo.tripkit.ui.geocoding

import com.skedgo.tripkit.ui.data.places.LatLngBounds
import org.immutables.value.Value.Immutable
import org.immutables.value.Value.Parameter

@Immutable(builder = false)
interface CitySelectedEvent {
    @Parameter
    fun bounds(): LatLngBounds?
}